cmp/internal/value/format.go

```diff
@@ -0,0 +1,258 @@
+// Copyright 2017, The Go Authors. All rights reserved.
```

Comment:
For GitHub, not that I know of.

For git, I think you can adjust the `
--find-renames[=<n>]` (aka `-M[<n>]`) flag of `git diff`, see https://git-scm.com/docs/git-diff. You may be able to get it to do what you want.

Alternatively, you can force seeing a diff between specific files by copying/pasting the two files into a diff tool, e.g., https://www.quickdiff.com/. (It'll look like [this](https://user-images.githubusercontent.com/1924134/28347042-c09f9170-6c02-11e7-99e9-ac1e71d6c20a.png).)